### DISCUSSION TITLE
Do you know how to disable warning at brew upgrade section when running topgrade?

### DISCUSSION THREAD

**Discussion by mio-dokuhaki** [topgrade-rs/topgrade#654] [Q&A] [answered]
While running topgrade, I get a warning "Warning: Calling ``brew upgrade --ignore-pinned`` is deprecated! There is no replacement."
Is there any way to disable showing it?
I have checked and there is no pinned formulae....

**Comment by SteveLauC** [comment, marked as answer]
Are you using the v14.0.0, this has already been fixed in #629

**Reply by mio-dokuhaki** [reply]
Really?
Then I'm sorry...
I am using MacOS and that version was still 13.0.0.
I will try to upgrade topgrade itself.
